pkg/model/lease.go

```diff
@@ -0,0 +1,89 @@
+package model
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Lease is a type corresponding to a Lease
+// table record
+type Lease struct {
+	AccountID                *string                 `json:"AccountId,omitempty" dynamodbav:"AccountId"`                                         // AWS Account ID
```

Comment:
I think we want the JSON stuff to mimic. 
https://github.com/Optum/dce/blob/master/pkg/api/response/lease.go#L26-L40

Dynamodbav stuff looks great

Reply:
fixed